src/transactions/feed/TransferFeedItem.tsx

```diff
@@ -71,17 +71,15 @@ function TransferFeedItem({ transfer }: Props) {
               style={[styles.amount, colorStyle]}
               testID={'TransferFeedItem/amount'}
             />
-            {!showTokenAmount && (
-              <TokenDisplay
-                amount={amount.value}
-                tokenId={amount.tokenId}
-                showLocalAmount={false}
-                showSymbol={true}
-                hideSign={true}
-                style={styles.tokenAmount}
-                testID={'TransferFeedItem/tokenAmount'}
-              />
-            )}
+            <TokenDisplay
+              amount={amount.value}
+              tokenId={amount.tokenId}
+              showLocalAmount={false}
+              showSymbol={true}
+              hideSign={true}
+              style={[styles.tokenAmount, { opacity: showTokenAmount ? 0 : 1 }]}
```

Comment:
i decided to hide this with opacity rather than not rendering the element because it keeps the layout better. i can also achieve this by making the amount container full height and aligning children to the top, but i think this is more brittle...